chall07/vde-dios.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+    "net/http"
+    "log"
+    "io/ioutil"
+    "fmt"
+    "regexp"
+    "strconv"
+    "strings"
+)
+
+func outputErr(err error){
+    if err != nil {
+        log.Fatalln(err)
+    }
+}
+
+func toHex(rgb [][]string) string {
+    output := ""
+    for i := 1; i < 4; i++ {
+        hex, err := strconv.Atoi(strings.Join(rgb[i],""))
+        outputErr(err)
+        output += fmt.Sprintf("%02x", hex)
+    }
+    return output
```

Comment:
As stated earlier, you could use something simpler and more readable to replace this function.

Reply:
📝